google/cloud/sql/connector/connector.py

```diff
@@ -70,10 +70,15 @@ class Connector:
         creating new event loop on background thread.
 
     :type sqladmin_api_endpoint: str
-    :param sqladmin_api_endpoint:
+    :param sqladmin_api_endpoint
         Base URL to use when calling the Cloud SQL Admin API endpoint.
         Defaults to "https://sqladmin.googleapis.com", this argument should
         only be used in development.
+
+    :type http_proxy: bool
+    :param http_proxy
+        Enables extracting proxy configuration from HTTP_PROXY, HTTPS_PROXY,
+        WS_PROXY or WSS_PROXY environment variables (all are case insensitive).
```

Comment:
nit: I believe a `enable_http_proxy` param name might convey better the idea that this is just a flag to allow for reading from the env vars. What do you think? (mentioning because when I first saw `http_proxy` I was expecting it to take the hostname value directly)